src/helm/proxy/clients/together_client.py

```diff
@@ -253,7 +253,7 @@ def do_it_sync() -> Dict[Any, Any]:
                 text=cleanup_str(raw_completion["text"], "together"),
                 logprob=sequence_logprob,
                 tokens=tokens,
-                finish_reason={"reason": raw_completion["finish_reason"]},
+                finish_reason={"reason": raw_completion.get("finish_reason")},
```

Comment:
Should `finish_reason` be `None` (no strong opinion)?

Reply:
Makes sense, I set `finish_reason` to `None` instead.